### PR TITLE
Added apply button and changed visuals for push plugin date-time-picker.

### PR DESCRIPTION
**Pull request by melihkorkmaz**
Added apply button and changed visuals for push plugin date-time-picker.

### CODE REVIEW

**Comment by ar2rsawseen** [comment]
Want @iartem to check it, when you are available :)

File: plugins/push/frontend/public/stylesheets/main.css [hunk 1]
```diff
@@ -987,7 +1015,7 @@ table.d-table .nothing { opacity: 0.3; }
 
 .comp-push input[type="number"]::-webkit-inner-spin-button,
 .comp-push input[type="number"]::-webkit-outer-spin-button {
-    -webkit-appearance:none;
```
Comment (iartem) [inline plugins/push/frontend/public/stylesheets/main.css]: This is rather wrong, we have number inputs not only in date picker

Comment (melihkorkmaz) [inline plugins/push/frontend/public/stylesheets/main.css]: Ok, I searched for other input-number related to .com-push class and I didn't find. 
But you're right. I can add a specific class for those inputs.

File: plugins/push/frontend/public/javascripts/component.datepicker.js [hunk 2]
```diff
@@ -82,17 +68,22 @@ window.component('datepicker', function(datepicker) {
 			m('.picker', [
 				m('.comp-datepicker-ui-picker', {config: datepicker.config(ctrl)}),
 				m('.comp-datepicker-time', [
-					m('span.comp-datepicker-time-label', 'Time'),
```
Comment (iartem) [inline plugins/push/frontend/public/javascripts/component.datepicker.js]: Let's localise this

File: plugins/push/frontend/public/stylesheets/main.css (hunk 1 above)
Comment (melihkorkmaz) [inline plugins/push/frontend/public/stylesheets/main.css]: Ok to be honest, I couldn't find a solution to return back input-number to browser defaults. I asked on stack-overflow, but nobody has answered it yet. https://stackoverflow.com/questions/45230816/browser-default-value-for-input-number-arrows

Comment (melihkorkmaz) [inline plugins/push/frontend/public/stylesheets/main.css]: did you have time to check functionality? I couldn't find an answer for the problem right above. So I can cancel to adding a spinner to input.

**Pull request closed without merging**